### PR TITLE
IntelliJ can handle Kotlin qualifiers

### PR DESCRIPTION
**Pull request by Leland-Takamine**
Update `IntelliJAnnotation.members` implementation to support Kotlin.

### CODE REVIEW

File: intellij/ast/src/main/kotlin/motif/ast/intellij/IntelliJAnnotation.kt [hunk 1]
```diff
@@ -43,18 +46,14 @@ class IntelliJAnnotation(
...
     }
 
     override val members: List<IrMethod> by lazy {
-        psiAnnotation.parameterList.attributes
-                .map {
-                    it.reference!!.resolve() as PsiMethod
-                }
-                .map { IntelliJMethod(project, it, PsiSubstitutor.EMPTY) }
+        val annotationClass = annotationClass ?: return@lazy emptyList<IrMethod>()
+        annotationClass.methods.map { IntelliJMethod(project, it, PsiSubstitutor.EMPTY) }
```
Comment (tyvsmith) [inline intellij/ast/src/main/kotlin/motif/ast/intellij/IntelliJAnnotation.kt]: possible NPE if annotationClass resolves to null after the elvis operator?

Comment (Leland-Takamine) [reply intellij/ast/src/main/kotlin/motif/ast/intellij/IntelliJAnnotation.kt]: This wouldn't compile if that was the case. The `?: return@lazy emptyList<IrMethod>()` is a quick return if `annotationClass` is null.

Comment (Leland-Takamine) [reply intellij/ast/src/main/kotlin/motif/ast/intellij/IntelliJAnnotation.kt]: The second reference to `annotationClass` is a reference to the local `val`, not the field.

Comment (tyvsmith) [reply intellij/ast/src/main/kotlin/motif/ast/intellij/IntelliJAnnotation.kt]: 👍

**Pull request merged**